### PR TITLE
Sixtrack5 docs update and minor build bug fix

### PR DESCRIPTION
**Pull request by vkbo**
Updates to README and to the user manual.

Also included here is a minor issue in the build system where the linking of LIBARCHIVE was missing for SixDA.

### CODE REVIEW

**Comment by vkbo** [comment]
Also added @rdemaria and @tpersson to the author list. This is not clear from the diff.

File: doc/user_manual/chIntroduction.tex [hunk 1]
```diff
@@ -103,9 +103,19 @@ \section{Evolution of SixTrack}
     \item \textbf{Version 4}
         \todo[inline]{Add stuff}
     \item \textbf{Version 5}
-        \todo[inline]{Add stuff}
+        Includes a full rewrite of the input parsing, upgrade of the source to Fortran 2008, and makes all major arrays dynamically allocatable.
```
Comment (kyrsjo) [inline doc/user_manual/chIntroduction.tex]: Put input parsing on a separate point, and not the very first thing. Also mention the splitting of the files, change to industry-standard preprosessing system, cmake.

Comment (kyrsjo) [reply doc/user_manual/chIntroduction.tex]: (I see astuce is mentioned already bellow, sorry I missed it)

File: doc/user_manual/chIntroduction.tex [hunk 2]
```diff
@@ -103,9 +103,19 @@ \section{Evolution of SixTrack}
...
+        Machine size and particle numbers are no longer defined by compiler flags.
```
Comment (kyrsjo) [inline doc/user_manual/chIntroduction.tex]: ... but the correct amount of memory is allocated on the fly. Arbitrary number of particles are supported.

**Comment by vkbo** [comment]
Suggestions implemented.
Someone more familiar with v4 should probably write that bit too for consistency.

File: doc/user_manual/chIntroduction.tex [hunk 3]
```diff
@@ -103,15 +103,15 @@ \section{Evolution of SixTrack}
...
-        Machine size and particle numbers are no longer defined by compiler flags.
-        Replaces the astuce preprocessor with the c preprocessor.
+        Machine size and particle numbers are no longer defined by compiler flags but allocated dynamically based on the input files.
         Merges the Collimation version of SixTrack into the main code.
-        Code wide implementation of rounding of input float values from text, as well as output of float values to text in critical parts, provided by the CRLIBM library.
+        Code-wide implementation of rounding of input float values from text, as well as output of float values to text in critical parts, provided by the CRLIBM library.
         This also includes a wrapper for consistent rounding of values from mathematical functions.
         Adds experimental support for HDF5 and ROOT output files.
         Extensive rewrite of many sections of the code to increase flexibility and modularity, as well as bug fixes to the physics.
+        Includes a full rewrite of the input parsing and upgrade of the source to Fortran 2008.
```
Comment (kyrsjo) [inline doc/user_manual/chIntroduction.tex]: This is really two quite separate points; I would also mentioned that with Fortran 2008 came modules!

File: doc/user_manual/chIntroduction.tex [hunk 4]
```diff
@@ -103,15 +103,15 @@ \section{Evolution of SixTrack}
...
         Support for 32, 64 and 128 bit floating point precision.
+        Replaces the astuce preprocessor with the c preprocessor.
```
Comment (kyrsjo) [inline doc/user_manual/chIntroduction.tex]: + split into multiple, smaller files.

**Comment by kyrsjo** [comment]
Some even more minor comments have arrived. Let's deal with v4 in a separate PR (possibly written over a beverage).

File: doc/user_manual/chIntroduction.tex (hunk 4 above)
Comment (vkbo) [reply doc/user_manual/chIntroduction.tex]: Too much detail for a summary in my opinion. No?

File: doc/user_manual/chIntroduction.tex (hunk 3 above)
Comment (vkbo) [reply doc/user_manual/chIntroduction.tex]: That existed long before 2008. And also existed in the code pre SixTrack 5.

**Comment by vkbo** [comment]
Moved the version into a module. been on my ToDo list for a while as the current implementation breaks the convention of having all included files in the include folder.

File: README.md [hunk 5]
```diff
@@ -7,8 +7,65 @@ It is mainly used for the LHC for dynamic aperture studies, tune optimization, a
...
+K.&nbsp;Sjobak
+
+**Core Devs:**
+A.&nbsp;Mereghetti,
+J.&nbsp;Molson,
+T.&nbsp;Persson
+
+## License
+
+Copyright 2014 CERN. This software is distributed under the terms of the GNU Lesser General Public License version 2.1, copied verbatim in the file "[COPYING](source/COPYING)".
```
Comment (kyrsjo) [inline README.md]: 2014 -> 2018?

**Pull request merged**